packages/blade/src/components/ActionList/ActionListBox.web.tsx

```diff
@@ -84,12 +84,11 @@ const useFilteredItems = (
 } => {
   const childrenArray = React.Children.toArray(children); // Convert children to an array
 
-  const { filteredValues, hasAutoCompleteInBottomSheetHeader, dropdownTriggerer } = useDropdown();
+  const { filteredValues, hasAutoCompleteInHeader, dropdownTriggerer } = useDropdown();
```

Comment:
can you comment for this ?
// now we can have autoComplete in DropDownHead , BottomSheetHeader. 

will help other devs / AI

Reply:
done. I added comment at place where this prop is defined so it shows up in jsdoc everywhere